src/main.rs

```diff
@@ -6,15 +6,23 @@ extern crate git2;
 use std::env;
 use std::path::Path;
 
+fn main2() {
```

Comment:
Rebase on top of issue 4 when that's done, and make sure that src/main.rs is all cleaned up afterwards.